week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,10 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var sum = 15 + 8;
+var percentageOfStudents = (15 / 23) * 100;
+var percentageOfMentors = (8 / 23) * 100;
```

Comment:
You should use the variables `numberOfStudents` and `numberOfMentors` here so that the calculation will still be valid if these numbers are changed. You always want to make your code as reusable as possible.